src/rws_interface.cpp

```diff
@@ -117,6 +117,111 @@ std::string RWSInterface::getIOSignal(const std::string& iosignal)
   return result;
 }
 
+bool RWSInterface::getMechanicalUnitStaticInfo(const std::string& mechunit, MechanicalUnitStaticInfo* p_static_info)
+{
+  bool result = false;
+
+  if (p_static_info)
```

Comment:
Suggestion: fail early.

```suggestion
  if (!p_static_info)
    return false;
```

Note: this suggestion does not apply cleanly (didn't want to make it as long as the entire method body). It's just to illustrate what I mean.

Reply:
Should not be needed anymore since I changed to passing references instead of raw pointers.